requirements/ci.txt

```diff
@@ -34,15 +34,15 @@ requests==2.28.1
     # via codecov
 six==1.16.0
     # via tox
-toml==0.10.2
+tomli==2.0.1
```

Comment:
https://github.com/tox-dev/tox/pull/2463/files
Change is coming from tox which started using `tomli` instead of toml for python<3.11